chore: added the extra examples from academy to the python readme

John-Memphis opened the pull request:
Search for "TO:DO" (no quotes) in the file. There's a part that needs to be double checked for clarity about what the words actually mean.

idanasulinmemphis commented:
@yanivbh1 please review

yanivbh1 commented on README.md:
```diff
@@ -93,7 +93,112 @@ if __name__ == '__main__':
+
+For details on deploying memphis open-source with different types of connections see the [docs](https://docs.memphis.dev/memphis/memphis-broker/concepts/security).
+ 
+The default connection type (for cloud) is using a JWT token-based connection. Here is an example of using memphis.connect with a JWT token:
```
@idanasulinmemphis @shay23b not sure I understand this comment. Also, the default connection method is password-based.

yanivbh1 replied:
@John-Memphis I dont see any example with `account id` for connecting with the Cloud

John-Memphis replied:
@yanivbh1 I'll add an example for that.
![image](https://github.com/memphisdev/memphis.py/assets/136013599/266e97b3-d2da-4959-91ae-166b3bd60988)
![image](https://github.com/memphisdev/memphis.py/assets/136013599/a34e67ce-61f0-4a47-ad04-e143596182ff)

All of the repos say that the default connection is token based. I've only been using Memphis with open-source and deployed with docker--where password is the default. I was assuming it was a kubernetes thing or a memphis cloud thing that used tokens by default.

If that's not the case I'll go and change that in all of the repos.

yanivbh1 commented on README.md:
```diff
@@ -104,64 +209,218 @@ await memphis.close()
-  partitions_number=1, # defaults to 1 
-  dls_station="<station-name>" # defaults to "" (no DLS station) - If selected DLS events will be sent to selected station as well
-)
+**A station will be automatically created for the user when a consumer or producer is used if no stations with the given station name exist.**<br><br>
```
@John-Memphis , please add a phrase on what a station is

John-Memphis replied:
👍

yanivbh1 reviewed (commented):
Comments inside

John-Memphis commented:
Synced with master, addressed comments from @yanivbh1 and changed some of the sentences throughout for a better reading experience. 

This readme update should be ready.

idanasulinmemphis commented:
@John-Memphis for me that looks ok, @yanivbh1 please take a look as well

yanivbh1 commented:
@idanasulinmemphis looks great to me
